pkg/skaffold/runner/runner.go

```diff
@@ -207,23 +207,41 @@ func (r *SkaffoldRunner) Dev(ctx context.Context, out io.Writer, artifacts []*v1
 	changed := changes{}
 	onChange := func() error {
 		logger.Mute()
+		unmute := true
```

Comment:
I would rename this to `hasError`, I think that would be more readable, what do you think?

Reply:
ping @dgageot ?

Reply:
Yes, `hasError` is much better! I though I pushed this fix but I must have lost it.